FMB/FiniteModelBuilder.cpp

```diff
@@ -413,7 +413,7 @@ void FiniteModelBuilder::init()
   if(env.options->fmbAdjustSorts() == Options::FMBAdjustSorts::PREDICATE){
     DArray<bool> deleted_functions(env.signature->functions());
     for(unsigned f=0;f<env.signature->functions();f++){
-      deleted_functions[f] =  (bool)_prb.getEliminatedFunctions().findPtr(f) || env.signature->getFunction(f)->usageCnt()==0;
+      deleted_functions[f] = env.signature->getFunction(f)->usageCnt()==0;
```

Comment:
Just a passing thought, as I have recently seen that usage counts are not updated for type constructors for example [here](https://github.com/vprover/vampire/blob/513a213c3a39720520eef6d1c02b6c744288031f/Shell/Property.cpp#L111). Is there a reason this (or any similar) operation is skipped for them?

Reply:
I think they come from an age when type constructors were not a thing. This does not matter much (at all) for FMB which (at least mentally) I keep compatible only with cnf/fof/tff. 

But it's a good point. And unused sort elimination is a valid (although trivial) model building technique.

Reply:
Sure, maybe they have no meaning here. Thanks for checking!